README.md

```diff
@@ -121,6 +121,7 @@ information on what to include when reporting a bug.
 
 ## Changelog
 
+ - [Win32] Added support for dark title bar #2228
```

Comment:
Should be inserted at the top of the other [win32] items in this last.

Reply:
@ws909 Done.